### ISSUE TITLE
Feature: Case-insensitive for column names with `LOAD FROM`

### ISSUE THREAD

**Issue by ray6080** [kuzudb/kuzu#3916]
### API

Other

### Description

Table and column names are case insensitive when it comes to Kuzu tables, however, we force case sensitive when it comes to `LOAD FROM` statements. I propose to change column names in `LOAD FROM` into case insensitive too.

**Comment by prrao87** [comment]
Yup, we would want to be similar in behaviour to DuckDB and Postgres, which are also case insensitive on table names.

**Comment by ray6080** [comment]
This actually also causes another problem in RDBMS extension. When we attach a postgres database, given a table with schema like this:
```sql
Comment(ID BIGINT, creationDate TIMESTAMP, locationIP varchar, browserUsed varchar, content varchar, length INT, PRIMARY KEY(ID))
```
We force users to `LOAD FROM` with all lower-case column names like this
```cypher
load from Comment return min(id), min(creationdate), min(locationip), min(browserused), min(content), min(length);
```
Otherwise will throw binder exceptions "Variable x is not in scope."
This is very confusing, as we changed the column names in users' perspective. I think this should be fixed in a high priority.

**Comment by andyfengHKU** [comment]
Fixed in #5200

**Comment by acquamarin** [comment]
Column names in postgres is case-insensitive. Postgres convertes all upper case column names to lower case.
